sdk/storage/Azure.Storage.Blobs/src/BlockBlobClient.cs

```diff
@@ -357,24 +358,19 @@ protected sealed override BlobBaseClient WithSnapshotCore(string snapshot)
         /// A <see cref="RequestFailedException"/> will be thrown if
         /// a failure occurs.
         /// </remarks>
+        [ForwardsClientCalls]
         public virtual Response<BlobContentInfo> Upload(
             Stream content,
             BlobHttpHeaders httpHeaders = default,
             Metadata metadata = default,
             BlobRequestConditions conditions = default,
             AccessTier? accessTier = default,
             IProgress<long> progressHandler = default,
-            CancellationToken cancellationToken = default) =>
-            UploadInternal(
-                content,
-                httpHeaders,
-                metadata,
-                conditions,
-                accessTier: accessTier,
-                progressHandler,
-                false, // async
-                cancellationToken)
-                .EnsureCompleted();
+            CancellationToken cancellationToken = default)
+        {
+            var uploader = new PartitionedUploader(this, default, BlockBlobMaxUploadBlobBytes);
+            return uploader.Upload(content, httpHeaders, metadata, conditions, progressHandler, accessTier, cancellationToken);
```

Comment:
NIT - for each parameter new line, same for other lines below

Reply:
fixed.